apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1505,6 +1506,24 @@ export class PortfolioService {
           subDays(new Date().setHours(0, 0, 0, 0), 1)
         ]);
         break;
+      case '5d':
```

Comment:
Is this the correct logic? I would rather expect it to be the range from Monday to today (aka _Week to date_).

Reply:
WTD is fair logic to have. But the way that Yahoo finance (a portfolio tracking system that I really like) implements it appears to be the last 5 trading days - so not exactly what is implemented here either.